### ISSUE TITLE
Discuss timing issues when publishing

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#1043]

**Comment by fernandolucchesi** [comment]
Is this the same as #1038? @nilsml

**Comment by nilsml** [comment]
I don't think it is exactly the same. This is more about having to wait to see the published results because of caching etc. This is "worse" than with AEM, so we should figure out exactly how this is working.

**Comment by wenche** [comment]
The current revalidate time is
`120 seconds` for all content coming from Sanity
`1800 seconds` (30 min) for the archived news
This means that for content that lives in Sanity, Next.js will try to re-generate the content at most once every 120 seconds.

**Comment by wenche** [comment]
We have adjusted the time to `60 seconds` for the Sanity content.

So at the moment, the rules are:
- For all content that lives in Sanity, Next.js will try to re-generate the content at most once every 60 second
- For the search, we will run the query once every 5 minutes (#833 for improvements)

It is important to understand that the timing is a balance between performance, user experience and cost.  It is also important to understand the model we are using. I can recommend this [article for a throughout explanation about ISR](https://www.smashingmagazine.com/2021/04/incremental-static-regeneration-nextjs/)

It is an option to improve more if this is not good enough. We have discussed approaches like
- Have different revalidate times for news articles and topic content
- Use [Next.js' new features for on-demand revalidation time](https://nextjs.org/blog/next-12-1#on-demand-incremental-static-regeneration-beta) for pages like the front page (for latest news component). Next.js will probably add more options to this in later releases
- Create some components, like for example promo cards, with SWR or Next.js new features
